src/components/OrganizationsList/OrganizationsList.tsx

```diff
@@ -1,20 +1,55 @@
+import React from 'react';
 import { OrgObj } from '@/api/cf/cloudfoundry-types';
-import { sortObjectsByParam } from '@/helpers/arrays';
-import { GridList } from '../GridList/GridList';
-import { OrganizationsListItem } from './OrganizationsListItem';
+import { chunkArray, sortObjectsByParam } from '@/helpers/arrays';
+import { CardRow } from '@/components/Card/CardRow';
+import { OrganizationsListCard } from './OrganizationsListCard';
+
+export function OrganizationsList({
+  orgs,
+  userCounts,
+  appCounts,
+  memoryAllocated,
+  memoryCurrentUsage,
+  spaceCounts,
+  roles,
+}: {
+  orgs: Array<OrgObj>;
+  userCounts: { [orgGuid: string]: number };
+  appCounts: { [orgGuid: string]: number };
+  memoryAllocated: { [orgGuid: string]: number };
+  memoryCurrentUsage: { [orgGuid: string]: number };
+  spaceCounts: { [orgGuid: string]: number };
+  roles: { [orgGuid: string]: Array<string> };
+}) {
+  if (!orgs.length) {
+    return <>no orgs found</>;
+  }
 
-export function OrganizationsList({ orgs }: { orgs: Array<OrgObj> }) {
   const orgsSorted = sortObjectsByParam(orgs, 'name');
+  const orgsGrouped = chunkArray(orgsSorted, 3);
```

Comment:
[non-blocking] I wonder if we could do this with CSS only? Right now at the tablet breakpoint we get some unexpected stacking behavior. We may need to adjust this so each row only contains two cards at the tablet breakpoint or look into an alternative approach. Let's tackle this in a different issue as to not block this work though.

![Screenshot 2024-10-17 at 1 15 28 PM](https://github.com/user-attachments/assets/22c9d86a-9315-4e2e-82cc-7d3e72fe262e)

Reply:
Ah I see, good catch. We may need to abandon the use of `grid-row`.

Reply:
I've made a new issue for this (#564)